README.md

```diff
@@ -676,6 +678,33 @@ As a convenience, the `id` and `itemToString` props which you pass to
 
 ## Event Handlers
 
+Downshift has a few events for which it provides implicit handlers. All of these
+handlers prevent the default event action, and a few of them have additional
+functionality.
+
+### default handlers
+
+* `ArrowDown`: moves the highlighted index down by 1. If this shift key is held
+  when this event fires, the highlighted index will jump down 5 indices instead of 1.
+  NOTE: if the current highlighed index is within the bottom 4 indices, the top-most
+  index will be highlighted.)
+
+* `ArrowUp`: moves the highlighted index up by 1. If this shift key is held when
+  this event fires, the highlighted index will jump up 5 indices instead of 1. NOTE:
+  if the current highlighed index is within the top 4 indices, the bottom-most index
+  will be highlighted.)
+
+* `Enter`: if the menu is open, select the currently highlighted item. NOTE: as
+  stated above, the usual 'Enter' event is prevented by Downshift's default implicit
+  enter handler, so, for example, a form submission event will not work as one
+  might expect. See below for customizing the handlers.
```

Comment:
This is only true for when the menu is open. If the menu is closed then `event.preventDefault()` will not be called and form submission should work.